release_notes.md

```diff
@@ -1,18 +1,55 @@
+## Advanced Playbook Generation
 
-> [!Note]
-> The *Voice Recognition* feature is currently unsupported on the Firefox browser as the webkit `SpeechRecognition` is not compatible with Mozilla Firefox. Hence, the mic button is not available when the FortiSOAR&trade; environment is accessed using the Firefox browser.
+FortiAI now produces more accurate, context-aware playbook blocks tailored to specific use cases. Whether automating standard workflows or designing complex incident response logic, it offers an efficient starting point that reduces manual effort.
+
+Key improvements include:
+
+- Playbook creation from simple prompts or clicks
+- Accurate detection of connector step types, operation names, and available inputs
+- Identification of steps that require loops for record traversal
+
+### Improved Jinja Expression Selection
+
+FortiAI intelligently selects appropriate Jinja expressions for each playbook step, reducing the need for manual adjustments.
+
+Structured outputs are now supported, with JSON data displayed in a readable JSON editor.
+
+## Configuration Wizard Enhancements
 
-**Conversation Thread Management**
+The configuration wizard has been improved to support the following:
 
-Previously, conversation threads were deleted from OpenAI whenever *Clear Conversation* was clicked, and a new thread was started with each new prompt.
+- Creation of SOC and Playbook assistants
+- Updating assistant instruction sets
+- Clearer error messages for scenarios such as:
+    - Insufficient permissions
+    - Failure during assistant creation or update
+    - Missing default connector configuration in single-user setups
+    - Unspecified user configuration in multi-user environments
 
-Now, threads are retained within OpenAI, enabling improved historical reference and continuity. Each new prompt, after clearing the conversation. still initiates a fresh thread, and the *Clear Conversation* option within the user interface continues to clear conversation history, ensuring a clean workspace.
\ No newline at end of file
+### AI Assistant Enhancements
+
+- **Attachment support**: Added support for file attachments (applies only to connector generation).
+
+- **Structured outputs**: JSON data is now displayed in a JSON editor for improved readability.
+
+- **Clear conversation**: The "Clear Conversation" feature is now available for both playbook and connector generation. It also clears conversation threads and any attached temporary files.
+
+- **Context-aware assistant loading**: The bot now detects the active page and loads the appropriate assistant automatically.  
+  - On the Content Hub "Create" page, the Connector Generation Assistant will be available.  
+  - On the Playbook Designer page, the Playbook Assistant will be loaded.
```

Comment:
Please also mention about SOC assistant as well, that it will be available on modules like Alerts, Incidents, etc.

Reply:
SOC assistant was always present and hence cannot be part of "What's new".